meetings/2021-10-05.md

```diff
@@ -0,0 +1,116 @@
+
+### IRI Prefix
+
+https://github.com/solid/data-interoperability-panel/issues/120
+
+eP: Originally this was posted for Data Registrations since this was
+the only thing applications were using. Now authorization agent creates
+different things in different places. Need to decided whether to use
+the same pattern in other places. First place is Access Consent Registry.
+Authorization Agent has to know how to create IRIs for them.
+
+JB: does the server need to understand iriPrefix?
+
+eP: Primarily used for client managed triples. Client has to understand
+how the server operates.
+
+JB: Confirming that iriPrefix would be equivalent to Data Registration in 
+conformant solid hierarchical containment. (eP: YES)
+
+eP: This is more for storages that don't work with hierarchical containment.
+Client relying on iriPrefix just needs to understand how to use the
+iriPrefix. The party adding the iriPrefix to the container 
+(e.g. the Authorization Agent) needs to know how the storage operates 
+(hierarchical namespace vs. flat namespace).
+
+JB: #120 - covers Data Registry, Agent Registry, Access Consent Registry?
+
+eP: Anything that we use container for.
+
+JB: would you expect that data itself would use iriPrefixes
+
+eP: good question - in that case it could be better as server managed. pragmatic way
+could be to put this on hold, and then push upstream to resolve on solid protocol
+level for how non-hierarchical containers can work. look at it on the storage
+level instead (so that server could manage it).
+
+### Update on Specification Progress
+
+https://github.com/solid/data-interoperability-panel/projects/1
+
+Closed: 
+
+* https://github.com/solid/data-interoperability-panel/issues/142
+* https://github.com/solid/data-interoperability-panel/issues/184
+* https://github.com/solid/data-interoperability-panel/issues/189
+* https://github.com/solid/data-interoperability-panel/issues/181
+* https://github.com/solid/data-interoperability-panel/issues/191
+* https://github.com/solid/data-interoperability-panel/issues/151
+* Editorial review and adjustment of shape trees
+
+
+JB: We defined Data Grants as immutable
+JB: We refactored Access Needs section. Especially Access Needs Decorators which provide i18n
+Barath: Is part of the confusion that decorator is used to mean something different?
+JB: The word is used in different contexts. I'm open to better naming.
+Barath: Tooltip is being used in UI for description of element.
+JB: In simplest sense it's access description or explanation.
+
+
+Current-focus / In-progress:
+
+* https://github.com/solid/data-interoperability-panel/issues/179
+
+Barath: I've looked at some inrupt docs on consent. I need to compare it with interop spec.
+eP: It would be great to start on Trusted Grants and delegation chains. This would
+take care of former `acl:Control` scenario.
+JB: I see various implications on requiring Write on the container to be able to create or update contained resources.
```

Comment:
```suggestion
Barath: I've looked at some inrupt docs on consent. I need to compare it with interop spec.

eP: It would be great to start on Trusted Grants and delegation chains. This would
take care of former `acl:Control` scenario.

JB: I see various implications on requiring Write on the container to be able to create or update contained resources.
```